cfe_internal/update/update_policy.cf

```diff
@@ -634,36 +634,36 @@ bundle agent modules_presence
 #              be updated in $(sys.workdir)/modules, including any sub-directories.
 {
   vars:
-    "vendored_dir" string => "$(this.promise_dirname)/../../modules/packages/vendored/";
```

Comment:
thank you!

Reply:
let's hope it works! :) (aka I did all the replacements right)

```
g/vendored_dir/s//_&/g
```

and similar.